_config.yaml

```diff
@@ -37,21 +39,32 @@ defaults:
       layout: post
 
 collections:
+  # generate page for each member
   # generate page for each member
   members:
     output: true
   # generate page for each post
   posts:
     output: true
+  # generate page for each post
+  posts:
+    output: true
 
+# jekyll plugins
 # jekyll plugins
 plugins:
+  - jekyll-spaceship
```

Comment:
I missed this. There's a bunch of duplication here that could be causing the build failures after you merged this.